Rakefile

```diff
@@ -7,6 +7,6 @@ require 'puppet-lint/tasks/puppet-lint'
 
 PuppetLint.configuration.send('disable_relative')
 
-task :task_acceptance => [:spec_prep, :beaker] do
+task :beaker => [:spec_prep] do
```

Comment:
Does this only add a dependency with `:beaker`? Or does it override the previous behavior?

Reply:
I think it overrides previous. Not sure how this is passing in travis... It is failing for https://jenkins-master-prod-1.delivery.puppetlabs.net/view/__experimental%20automatic/job/experimental_auto_donoghuc-facts_intn-sys_smoke-gem-bolt/2/PLATFORM=debian8-64default.a,WORKER_LABEL=beaker/console. Will work on solution tomorrow. :disappointed: